src/main/java/seedu/address/logic/commands/task/note/ClearNoteCommand.java

```diff
@@ -6,31 +6,34 @@
 
 import seedu.address.logic.commands.Command;
 import seedu.address.logic.commands.CommandResult;
-import seedu.address.model.AddressBook;
 import seedu.address.model.Model;
-import seedu.address.model.todo.Note;
+import seedu.address.model.NoteList;
+import seedu.address.model.tag.TodoType;
+import seedu.address.model.task.Note;
 
 /**
  * Clears the address book.
  */
 public class ClearNoteCommand extends Command {
 
-    public static final String COMMAND_WORD = "clear note";
+    public static final String COMMAND_WORD = "clear_note";
```

Comment:
I think having an underscore making COMMAND_WORD into a single word is a good decision and should be applied to all commands also. 👍